hazelcast-jet-distribution/pom.xml

```diff
@@ -208,6 +208,18 @@
             <version>${project.version}</version>
             <classifier>jar-with-dependencies</classifier>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast.jet</groupId>
+            <artifactId>hazelcast-jet-cdc-debezium</artifactId>
+            <version>${project.version}</version>
+            <classifier>jar-with-dependencies</classifier>
+        </dependency>
+        <dependency>
+            <groupId>com.hazelcast.jet</groupId>
+            <artifactId>hazelcast-jet-cdc-mysql</artifactId>
+            <version>${project.version}</version>
+            <classifier>jar-with-dependencies</classifier>
```

Comment:
how big is each JAR? would it make sense to require two JARs for MySQL, rather than duplicating the content in each?

Reply:
12M (debezium) & 18M (mysql), quite similar to the other ones: 14M (grpc), 14M (python), 10M (s3); it's convenient to need a single jar, but it's not that big of a deal; should I remove the duplication or leave it?

Reply:
Let's leave it for now, but we may revisit in the future.

Reply:
Actually... we will have at least 2 more like the mysql one (postgres, sqlserver). Unlikely that we're going to leave them like this. Maybe bite the bullet and do it now?

Reply:
Done, mysql jar is now 6.8K (and works).